pytest/backend/test_states_polyquad.py

```diff
@@ -399,6 +403,7 @@ class TestMultiModePolyQuadratureExpectations:
     def test_three_mode_arbitrary(self, setup_backend, pure, hbar, tol):
         """Test that the correct result is returned for an arbitrary quadratic polynomial"""
         backend = setup_backend(3)
+        # increase the cutoff to 7 for accuracy
```

Comment:
I don't think this should be hidden deep inside the test like this

Reply:
Okay, do you think it's fine to have two magic numbers for cutoff at the top of the file?

Reply:
yup, as long as they are clearly named wrt which tests they apply to